packages/components/menu/src/menu-label.ts

```diff
@@ -0,0 +1,36 @@
+import { defineComponent, h, mergeProps, reactive } from 'vue'
+import { Primitive, primitiveProps } from '@oku-ui/primitive'
+import { reactiveOmit, useForwardRef } from '@oku-ui/use-composable'
+import type { MenuLabelNativeElement } from './props'
+import { MENU_LABEL_NAME, menuLabelProps, scopedMenuProps } from './props'
+
+const menuLabel = defineComponent({
+  name: MENU_LABEL_NAME,
+  inheritAttrs: false,
+  props: {
+    ...menuLabelProps.props,
+    ...primitiveProps,
+    ...scopedMenuProps,
+  },
+  emits: menuLabelProps.emits,
+
+  setup(props, { attrs, slots }) {
+    // const { scopeOkuMenu } = toRefs(props)
+
+    // const _reactive = reactive(menuLabelProps)
+    // const reactiveMenuLabelProps = reactiveOmit(_reactive, (key, _value) => key === undefined)
```

Comment:
When do we use them, or do we use them in all components?

Reply:
We use it everywhere with all ... written.